ImperatorToCK3/Source/Imperator/Provinces/Pops.h

```diff
@@ -10,26 +12,29 @@ namespace Imperator
 	public:
 		void loadPops(std::istream& theStream);
 
+		Pops& operator= (const Pops& obj) { this->pops = obj.pops; return *this; }
+
 		[[nodiscard]] const auto& getPops() const { return pops; }
 
 	private:
 		void registerKeys();
 
+		Pop::Factory popFactory;
+
 		std::map<unsigned long long, std::shared_ptr<Pop>> pops;
 	};
 
 
 	class PopsBloc : commonItems::parser
 	{
 	public:
-		PopsBloc() = default;
-		explicit PopsBloc(std::istream & theStream);
+		explicit PopsBloc(std::istream& theStream);
 
 		[[nodiscard]] const auto& getPopsFromBloc() const { return pops; }
 
 	private:
 		void registerKeys();
-
+		
```

Comment:
Spurious tabs.

Reply:
There is indentation in empty lines across the project, so this is consistent (not sure if I or the IDE added missing tabs here).